snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -113,11 +113,14 @@ type Bootstrapper struct {
 
 	// Called when bootstrapping is done on a specific chain
 	onFinished func(ctx context.Context, lastReqID uint32) error
+
+	nonVerifyingParser block.Parser
 }
 
 func New(config Config, onFinished func(ctx context.Context, lastReqID uint32) error) (*Bootstrapper, error) {
 	metrics, err := newMetrics(config.Ctx.Registerer)
 	return &Bootstrapper{
+		nonVerifyingParser:          config.NonVerifyingParse,
```

Comment:
This is intentional - the field is: `nonVerifyingParser block.Parser` which is a field of an interface. 
However, `config.NonVerifyingParse` is a function pointer `NonVerifyingParse block.ParseFunc` and therefore it is named as an action.

Reply:
When you use a function pointer, you apply it on an object, so it would be weird to call the function pointer reference a parser. It should be named as an action - `parse(x)` instead of `parser(x)`.

Reply:
Ah - I thought `NonVerifyingParse` was of type `block.Parser`. sgtm